### ISSUE TITLE
Map Search Radiua

### ISSUE THREAD

**Issue by PietBlootvoet** [osmandapp/OsmAnd#19903]
### Describe the idea (required)

Please remove the default search radius of 10km.  I tried to search within 1080km from Agulhas, but the map kept reverting to 10km.  There's a bug to be fixed. 
Please add the option to bypass the 10km limit. 

### Tell us about the expected behaviour (required)

Map search reverts to 10km even when set to go beyond 500km. Annoying

### Tell us about alternatives you've considered (required)

I have aphasia and it is hard to navigate the program anyhow. 

### Context (optional)

As discussed with Yurii

**Comment by DmitryAlexei** [comment]
@PietBlootvoet Have you used search as described in the documentation? Please see https://osmand.net/docs/user/search/search-address/
Search radius depends on the current map center. So if you are searching for something in Agulhas, it's better to center map on this city and perform search from there.

If the issue persists, please describe it more clearly (with step-by-step instructions or short video).

**Comment by PietBlootvoet** [comment]
As stated before, I suffer from various disabilities, one of which is aphasia.  Now Google that to see what it means!!  To bomb me with TMI isn't going to help.

**Comment by vshcherb** [comment]
Yes we should implement smart Auto increase. We can't set default > 10 km up to 500 km , it will lead to crash in many search situations
